main.tscn

```diff
@@ -1,12 +1,15 @@
-[gd_scene load_steps=12 format=3 uid="uid://7t8yksj1dbhf"]
+[gd_scene load_steps=15 format=3 uid="uid://7t8yksj1dbhf"]
 
 [ext_resource type="Script" path="res://main.gd" id="1_tw7s1"]
 [ext_resource type="PackedScene" uid="uid://yyaj3k1cmcfm" path="res://player.tscn" id="1_vm42q"]
 [ext_resource type="PackedScene" uid="uid://j47r672at4fb" path="res://obstacle_wall.tscn" id="2_dpaam"]
 [ext_resource type="Texture2D" uid="uid://dy2doeee2u4of" path="res://art/1x1_#FFFFFFFF.png" id="4_0dutb"]
 [ext_resource type="PackedScene" uid="uid://x3oe10u0lnm6" path="res://hud.tscn" id="4_8fpnk"]
+[ext_resource type="Script" path="res://moving_background.gd" id="5_0c8bv"]
 [ext_resource type="AudioStream" uid="uid://d5m5epf1lwu8" path="res://sound/fish1_dead.wav" id="6_0uv8k"]
-[ext_resource type="PackedScene" uid="uid://betx7xg7eqa2d" path="res://score_http.tscn" id="7_ib5i2"]
+[ext_resource type="Texture2D" uid="uid://cd6o37idatsij" path="res://art/water-1-repeat.png" id="6_s345y"]
+[ext_resource type="PackedScene" path="res://score_http.tscn" id="7_ib5i2"]
+[ext_resource type="Texture2D" uid="uid://bhh11byqbwupu" path="res://art/water-1.png" id="8_og6te"]
```

Comment:
My editor has issues trying to load this resource

Reply:
fixed